File: day 80/classwork/code/codewars3.js
// https://www.codewars.com/kata/527e4141bb2ea5ea4f00072f/train/javascript

function twistedSum(n) {
  let total = 0;

  for (let i = 1; i <= n; i++) {
    let numStr = i.toString(); 
    for (let num = 0; num < numStr.length; num++) {
      total += Number(numStr[num]);
    }
  }

  return total;
  }

File: Day 052/homework/code/codewars3.py
# https://www.codewars.com/kata/576bb71bbbcf0951d5000044/train/python
def count_positives_sum_negatives(arr):
    if arr is None or len(arr) == 0:  
        return []
    
    count_positives = 0
    sum_negatives = 0
    
    for num in arr:
        if num > 0:
            count_positives += 1
        elif num < 0:
            sum_negatives += num
            
    return [count_positives, sum_negatives]
